learn/react-native/mobile-build-manual.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+---
+---
+
+In application development process, you create iOS Package App Store (IPA) or Android Package Kit (APK) files for developing any mobile application. IPA file is the compressed version of the required files and resources of IOS application, created to test the IOS application before it is live. APK file is the similar package file used in Android devices.
+
+With React Native CLI, a built-in feature, you can create and manage your project on local platforms. You can build apk or ipa on your system from a react-native zip. To help the WaveMaker developers in this manual process, ```wm-reactnative-cli``` is created. ```wm-reactnative-cli``` combines multiple react-native commands into a single command. Please follow the below procedure.
```

Comment:
With React Native CLI, you can build the apk or ipa on your CI/CD machines easily.